File: uri/salario/salario.scala
object Main{
    def foo(tempo: Int, salario: Double) : Double =
        tempo * salario

    def main(args: Array[String]){
        var num = scala.io.StdIn.readInt() : Int
        var tempo = scala.io.StdIn.readInt() : Int
        var salario = scala.io.StdIn.readDouble() : Double
        
        var resultado = foo(tempo, salario)
        printf("NUMBER = %d\nSALARY = U$ %.2f\n", num, resultado );
    }
}

File: uri/salario/salario.ml

let calculafoo (x : int) (y : float) : float =
    ( float_of_int x ) *. y;;

let num = read_int();;
let b = read_int();;
let c = read_float();;
let foo = calculafoo b c;;
Printf.printf "NUMBER = %d\nSALARY = U$ %.2f\n" num foo;;